version

```diff
@@ -1 +0,0 @@
-0.1.3-SNAPSHOT
```

Comment:
how was this used before? did the helm-wrapper executable know its own version?

Reply:
This file was added by Philipp, https://github.com/evoila/helm-wrapper/commit/25dd956bbd4bef8fd6820cedc6594a59178ed123 in order to set image versions in the pipelines